addons/web/static/src/core/utils/hooks.js

```diff
@@ -46,7 +46,7 @@ export function useAutofocus({ refName, selectAll, mobile } = {}) {
         return ref;
     }
     // LEGACY
-    if (isMobileOS()) {
+    if (!mobile && isMobileOS()) {
```

Comment:
Strange to see it flagged `LEGACY` when `isMobileOS` is not legacy. `comp.env.isSmall` above is actually legacy while not being flagged

Reply:
and they are doing the same thing but !mobile is missing....